[C#] Code Snippet for Atlas Search Autocomplete doesn't work (#177)
mongodb/docs-csharp: source/fundamentals/atlas-search.txt

   :language: csharp
   :dedent:

> .. note::

Reviewer: S: add newline after directive

>    If the field you are searching has a search index, it must be included in your ``Autocomplete`` call. If a search index does not exist, the default index is used.

Reviewer: ```suggestion
   If the field you are searching on is indexed by a search index, you must pass the index name to the
   ``Autocomplete`` call. If a search index does not exist, the search uses the default index.
```

>    ``Search.Autocomplete(g => g.Make, "Gib"), indexName: <make_search_index>)``

Reviewer: Q: Is this issue unique to the `Autocomplete()` Search method? Can you please check with the Pm to verify if all of the code examples used on this page need to include the search index name?

Author: [According to the PM](https://jira.mongodb.org/browse/DOCSP-37267?focusedCommentId=6149483&focusedId=6149483&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6149483), yes, this is specific to Autocomplete and he did test all the methods except the Geo ones.

Reviewer: After reading the code example, I feel like the actual code example used in this section should be updated as well. The note could clarify that compared to the other examples, where the `default` index is used in the absence of the index name, this example explicitly requires the index name.
